module/VuFind/src/VuFind/RecordDriver/DefaultRecord.php

```diff
@@ -1311,6 +1315,28 @@ public function getUPC()
         return (array)($this->fields['upc_str_mv'] ?? []);
     }
 
+    /**
+     * Get UUIDs (Universally unique identifier).
+     *
+     * @return array
+     */
+    public function getUuid()
```

Comment:
Should this be getUuids since it may return more than one?

Reply:
I changed it